data/korath/korath missions.txt

```diff
@@ -61,3 +61,353 @@ mission "Discovered Korath Space"
+			choice
+				`	"I'm happy to help in whatever way I can."`
+				`	"You're welcome. Do you know how long it will take for them to make a working translator?"`
+			`	Rek translates your acknowledgement to Syciil, then continues speaking to you. "I spoke with the technologists after you dropped off the information. They say it will take some time to convert the Hai technology and methodology into something we would be able to use. It would then take even longer to program two new languages into the translator. From what I understand, it's too early to tell if the translator's programming would even work with Wanderer or Korath languages. I'll have to start working with them to create a database of the Korath language soon.`
+			`	"All things considered, the device probably won't be done for a long time - if ever - but it's a big step in the right direction. We have you to thank for that. That brings me to my next point: I'm sure you're curious about what the new part of the spaceport is. Please, follow me." You, Rek, and Syciil walk to the newly-developed area.`
+			`	Its centerpiece is several large, two-sided monitors. Each displays a map of local Korath space with settlements and hyperspace lanes labeled. Syciil holds her hand up to one, and a few different listings appear. Each shows the destination and displays the number of passengers and cargo. The most intuitive aspect of the setup is that it's entirely visual: there are no numbers or writing anywhere.`
+			`	Syciil speaks through Rek. "She says, 'A translation device is not the only thing Rek and I have been developing. We've been brainstorming a way for our populations to communicate without translators should the project fall through, and this is what we've come up with. The Wanderers were kind enough to lend us the resources to build these - though it took some deliberation to determine it would not be an insult to Korath honor to accept the help. Similar terminals were recently installed in other Kor Efret systems and are being built on the new Wanderer colonies as we speak. You are welcome to use these terminals as well, Captain <last>.'" She holds her hands up, palms forward, says a few words, then departs. "She has other matters to attend to," Rek explains. "In fact, I do as well. Thanks again for all your help, <first>. The Wanderers and Kor Efreti will both benefit from it, and I hope these new jobs will work to your advantage as well. Until next time," Rek says, then she leaves to catch up with Syciil. You're left to explore the new jobs available in Korath space.`
+			
+event "wanderers taking jobs from kor efreti"
```

Comment:
A small fleet has been added. I'll keep this conversation open in case others want to comment on whether they think it's necessary/too Act 3-impacting.